### PR TITLE
chore: change to coordinator layout 

### LINKED ISSUE #789: change to coordinator layout
**Describe the chore**

For #788 and #785 There is a need to change some layouts to coordinator layout

**Would you like to work on the issue?**

- [x] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by GOVINDDIXIT**
Fix #789

Some layouts are changed to coordinator layout, required for #788

### CODE REVIEW

**Comment by GOVINDDIXIT** [comment]
@iamareebjamal  pls review

File: app/src/main/res/layout/fragment_login.xml [hunk 1]
```diff
@@ -6,147 +6,154 @@
     android:fillViewport="true"
     android:fitsSystemWindows="true">
 
-    <LinearLayout
+    <androidx.coordinatorlayout.widget.CoordinatorLayout
```
Comment (iamareebjamal) [inline app/src/main/res/layout/fragment_login.xml]: Change the root element

File: app/src/main/res/layout/fragment_login.xml [hunk 2]
```diff
@@ -1,159 +1,154 @@
...
     xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/loginCoordinatorLayout"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:fillViewport="true"
     android:fitsSystemWindows="true">
 
-    <androidx.coordinatorlayout.widget.CoordinatorLayout
-        android:id="@+id/loginCoordinatorLayout"
+    <LinearLayout
```
Comment (iamareebjamal) [inline app/src/main/res/layout/fragment_login.xml]: Should be scroll view

File: app/src/main/res/layout/fragment_login.xml [hunk 3]
```diff
@@ -149,4 +151,4 @@
         </LinearLayout>
 
     </LinearLayout>
```
Comment (iamareebjamal) [inline app/src/main/res/layout/fragment_login.xml]: You replaced a ScrollView, so this should be wrapped in ScrollView or NestedScrollView to be specific

**Pull request merged**